frontend/src/basic/MainDialogs/index.tsx

```diff
@@ -56,7 +45,7 @@ const MainDialogs = (): JSX.Element => {
       />
       <AboutDialog
         open={open.info}
-        maxAmount={maxAmount}
+        maxAmount={'100000'} //FIXME About dialog shoul allow to change coordinator
```

Comment:
⚠️

Reply:
About dialog has to change for the federation. It can be simplified, as every coordinator has its own terms.